ldk-server-protos/src/proto/types.proto

```diff
@@ -683,3 +683,24 @@ message Bolt11InvoiceDescription {
     string hash = 2;
   }
 }
+
+// Configuration options for payment routing and pathfinding.
+// See https://docs.rs/lightning/0.2.0/lightning/routing/router/struct.RouteParametersConfig.html for more details on each field.
+message RouteParametersConfig {
+  // The maximum total fees, in millisatoshi, that may accrue during route finding.
+  // Defaults to 1% of the payment amount + 50 sats
+  optional uint64 max_total_routing_fee_msat = 1;
+
+  // The maximum total CLTV delta we accept for the route.
+  // Defaults to [`DEFAULT_MAX_TOTAL_CLTV_EXPIRY_DELTA`].
+  uint32 max_total_cltv_expiry_delta = 2;
+
+  // The maximum number of paths that may be used by (MPP) payments.
+  // Defaults to [`DEFAULT_MAX_PATH_COUNT`].
```

Comment:
The `defaults to` text here won't really work because it'll put the constant but not actually render/fetch the value. Will need to just put the actual value here sadly